bridge/opencensus/example_test.go

```diff
@@ -0,0 +1,42 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package opencensus_test
+
+import (
+	octrace "go.opencensus.io/trace"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/bridge/opencensus"
+	"go.opentelemetry.io/otel/sdk/metric"
+)
+
+func ExampleNewTracer() {
+	// Create an OpenTelemetry Tracer to use to record spans.
+	tracer := otel.GetTracerProvider().Tracer("go.opentelemetry.io/otel/bridge/opencensus")
+	// Overwrite the OpenCensus DefaultTracer so that it uses OpenTelemetry
+	// rather than OpenCensus.
+	octrace.DefaultTracer = opencensus.NewTracer(tracer)
+}
+
+func ExampleNewMetricProducer() {
+	// Create the OpenCensus Metric bridge.
+	bridge := opencensus.NewMetricProducer()
+	// Add the bridge to your reader using metric.WithProducer().
+	// If using a push exporter, such as OTLP, use metric.NewPeriodicReader.
+	// If using the prometheus exporter, use prometheus.WithProducer.
```

Comment:
I think something is wrong here (at least for me it is hard to follow). How about something like

```suggestion
	// Add the bridge as a producer to your reader.
	// If using a push exporter, such as OTLP exporter,
	// use metric.NewPeriodicReader with metric.WithProducer option.
	// If using a pull exporter which acts as a reader, such as prometheus exporter,
	// use a dedicated option like prometheus.WithProducer.
```